### PR TITLE
Revert New Learn Design and Title Styles from 2201.1.0

### PR DESCRIPTION
**Pull request by praneesha**
## Purpose
Revert the new learn page design and title style changes from 2201.1.0.
> Fixes #

## Checklist

- [ ] **Page addition**
  - [ ] Add `permalink` to pages.

- [ ] **Page removal**
  - [ ] Remove entry from corresponding left nav YAML file.
  - [ ] Add `redirect_from` on the alternative page.
  - [ ] If no alternative page, add redirection on the `redirections.js ` file.

- [ ] **Page rename**
  - [ ] Add front-matter `redirect_from`.
  - [ ] Add front-matter `redirect_to:` (if applicable).

- [ ] **Page restrcuture**
  - [ ] Add `permalink` to pages.
  - [ ] Add front-matter `redirect_from`.
  - [ ] Add front-matter `redirect_to:` (if applicable).

### CODE REVIEW

File: _data/ballerinaopenapisupportswanlake.yml [hunk 1]
```diff
@@ -1,39 +1,39 @@
-- title: Ballerina OpenAPI support
+- title: OpenAPI Tool
```
Comment (dakshitha) [inline _data/ballerinaopenapisupportswanlake.yml:1]:
```suggestion
- title: OpenAPI tool
```

File: _data/ballerinaasyncapisupportswanlake.yml [hunk 2]
```diff
@@ -1,6 +1,6 @@
-- title: Ballerina AsyncAPI support
+- title: AsyncAPI Tool
```
Comment (dakshitha) [inline _data/ballerinaasyncapisupportswanlake.yml:1]:
```suggestion
- title: AsyncAPI tool
```

File: _data/ballerinaopenapisupportswanlake.yml (hunk 1 above)
Comment (dakshitha) [inline _data/ballerinaopenapisupportswanlake.yml:1]:
```suggestion
- title: OpenAPI tool
```

File: learn/guides/asyncapi-tool.md [hunk 3]
```diff
@@ -1,13 +1,15 @@
 ---
 layout: ballerina-asyncapi-support-left-nav-pages-swanlake
-title: Ballerina AsyncAPI support 
+title: AsyncAPI sTool
```
Comment (dakshitha) [inline learn/guides/asyncapi-tool.md:3]:
```suggestion
title: AsyncAPI tool
```

File: learn/guides/openapi-tool.md [hunk 4]
```diff
@@ -1,10 +1,10 @@
 ---
 layout: ballerina-openapi-support-left-nav-pages-swanlake
-title: Ballerina OpenAPI support 
+title: OpenAPI Tool
```
Comment (dakshitha) [inline learn/guides/openapi-tool.md:3]:
```suggestion
title: OpenAPI tool
```

File: _data/ballerinaopenapisupportswanlake.yml (hunk 1 above)
Comment (praneesha) [reply _data/ballerinaopenapisupportswanlake.yml]: Fixed.

**Pull request merged**